fcomm/examples/Makefile

```diff
@@ -16,29 +16,41 @@ fibonacci-proof2.json : fibonacci-claim.json
 fibonacci-proof : fibonacci-proof.json
 fibonacci-proof2 : fibonacci-proof2.json
 
-verify-fibonacci-proof :
+verify-fibonacci-proof : fibonacci-proof.json
 	cargo run --release -- verify --proof fibonacci-proof.json
 
-num-list-commitment.json : num-list-function.json
+verify-fibonacci-proof2 : fibonacci-proof2.json
+	cargo run --release -- verify --proof fibonacci-proof2.json
+
+num-list-commitment : num-list-function.json
 	cargo run --release -- commit --function num-list-function.json --commitment num-list-commitment.json
 
-num-list-input.lurk : num-list-input.lurk
+num-list-opening : num-list-opening.json
 
-num-list-opening.json : num-list-function.json num-list-input.lurk
+num-list-opening.json : num-list-function.json num-list-input.lurk num-list-commitment
 	cargo run --release -- open --function num-list-function.json --input num-list-input.lurk --proof num-list-opening.json
 
-num-list-proof : num-list-opening.json
-
-verify-num-list-opening :
+verify-num-list-opening : num-list-opening.json
 	cargo run --release -- verify --proof num-list-opening.json
 
-chained-commitment.json : chained-function.json
+chained-commitment : chained-function.json
 	cargo run --release -- commit --function chained-function.json --commitment chained-commitment.json
 
 chained-opening: chained-opening.json
 
-chained-opening.json : chained-function.json chained-input.lurk
+chained-opening.json : chained-function.json chained-input.lurk chained-commitment.json
 	cargo run --release open --function chained-function.json --input chained-input.lurk --proof chained-opening.json --chain
 
-verify-chained-opening :
+verify-chained-opening : chained-opening.json
 	cargo run --release -- verify --proof chained-opening.json
+
+chained-opening2: chained-opening2.json
+
+chained-opening2.json : chained-opening.json
+	# NOTE: This concrete commitment (39ceb60c198b6b7c9f1ff98dd83026bd282e32307d8bd10a28f19d637c4d8549) can be found
+	# in chained-opening2.json, but its identity is determistic based on the secret and 'second return value' (cdr)
+	# of the function specified in chained-function.json.
+	cargo run --release open --commitment 39ceb60c198b6b7c9f1ff98dd83026bd282e32307d8bd10a28f19d637c4d8549 --input chained-input.lurk --proof chained-opening2.json --chain
+
+verify-chained-opening2 : chained-opening2.json
+	cargo run --release -- verify --proof chained-opening2.json
```

Comment:
It would be great to have one rule that exemplifies the use of the ` —request` argument to the `open` command. We had those with *-open-request.json, removed in this PR.

Reply:
Ah, fair enough. I was too zealous in my housecleaning. We can add it back.

Reply:
Added an example using the chained opening, since it requires hard-coding the commitment as above (in the request).